tests/utils.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+import time
+import kubernetes as k8s
+
+
+def wait_for_deployments_ready(e2_test_case,
+                               api: k8s.client.AppsV1Api,
+                               core_api: k8s.client.CoreV1Api,
```

Comment:
The `core_api` is never used in this method ?

Reply:
good catch!